### PR TITLE
new icon: framermotion (original, original-wordmark)

### LINKED ISSUE #1643: [ICON REQUEST]: framermotion
### I have searched through the issues and didn't find my problem.

- [x] Confirm

### About the icon

Framer motion is a powerful react animation library

### Links and sources

[https://www.npmjs.com/package/framer-motion](https://www.npmjs.com/package/framer-motion)
[https://github.com/framer/motion#readme](https://github.com/framer/motion#readme)

This is the [logo they use on their official documents,](https://www.framer.com/motion/) and the only OEM version of the framer motion logo I could find in SVG.
```
<svg xmlns="http:www.w3.org/2000/svg" viewBox="0 0 14 21" role="presentation">
    <path d="M0 0h14v7H7zm0 7h7l7 7H7v7l-7-7z" fill="currentColor"></path>
</svg>
```

The wordmark portion of the logo uses the **GT Walsheim** font and doesn't have an SVG format.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Gautam25Raj**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1643**

## Link to prove your SVG is correct and up-to-date.
https://www.framer.com/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -3146,6 +3146,31 @@
...
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original",
+                "original-wordmark"
+            ]
+        },
+        "color": "#000",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:3172]: You may want to add aliases here. 🙂

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Gautam25Raj — COMMENTED** [review]
I don't know how aliases work, but I guess this should work.

File: devicon.json [hunk 2]
```diff
@@ -3146,6 +3146,36 @@
...
+                "original-wordmark"
+            ]
+        },
+        "color": "#000",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "original-wordmark"
+            }
+        ]
```
Comment (lunatic-fox) [inline devicon.json:3177]: The idea behind aliases works like:
- By default an icon is an `original` version.
- Here is a flowchart about the possible bases and aliases:
```mermaid
flowchart LR;
original --> original-wordmark --> plain-wordmark
original-wordmark --> line-wordmark
original --> plain
original --> line
```
- Remember that the `base` is always an existent icon.
- In this case we have `original` and `original-wordmark`, both working as plain icons, solid colors and one path. Therefore, we can from the `base: original` make an `alias: plain`, same goes to `base: original-wordmark` to `alias: plain-wordmark`.
- Here is an example of complex aliases: #1465

```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            },
            {
                "base": "original-wordmark",
                "alias": "plain-wordmark"
            }
        ]
```

**Review by lunatic-fox — CHANGES_REQUESTED** [review]

File: devicon.json (hunk 2 above)
Comment (Panquesito7) [reply devicon.json:3177]: Nice chart! 🚀 We should probably save that in the Wiki.

**Review by Gautam25Raj — COMMENTED** [review]
I guess everything looks fine.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts [**here**](https://github.com/devicons/devicon/pull/1788/conflicts). If you need any help, you can ask for help here or on our [**Discord server**](https://discord.gg/hScy8KWACQ). You can also check the [GitHub Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) to make the process easier. 🙂

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🚀

**Pull request closed without merging**